pkg/authenticator/hash/scrypt_test.go

```diff
@@ -3,28 +3,43 @@ package hash
 import (
 	"strings"
 	"testing"
-
-	"github.com/stretchr/testify/assert"
 )
 
-var hasher = NewSCrypt()
-
-func TestBasicHash(t *testing.T) {
-	secretKey := "hello world"
-	hash, err := hasher.CreateHash(secretKey)
-	assert.Nil(t, err)
-	assert.NotNil(t, hash)
-
-	assert.Nil(t, hasher.VerifyHash(hash, secretKey))
-	assert.NotNil(t, hasher.VerifyHash(hash, "goodbye"))
-}
-
-func TestLongKey(t *testing.T) {
-	secretKey := strings.Repeat("A", 720)
-	hash, err := hasher.CreateHash(secretKey)
-	assert.Nil(t, err)
-	assert.NotNil(t, hash)
-
-	assert.Nil(t, hasher.VerifyHash(hash, secretKey))
-	assert.NotNil(t, hasher.VerifyHash(hash, secretKey+":wrong!"))
+func TestSCrypt_VerifyHash(t *testing.T) {
+	type args struct {
+		secretKey string
+	}
+	tests := []struct {
+		name     string
+		args     args
+		setup    func() error // Optional, delete if unused
```

Comment:
Should these be deleted?

Reply:
There is another ref below to these unused functions.

Reply:
Yes they should!